mkimage.sh

```diff
@@ -100,6 +100,12 @@ ENV DEBIAN_FRONTEND noninteractive
 LABEL io.resin.architecture="rpi" \
 	  io.resin.qemu.version="$RESIN_QEMU_VERSION" \\
 	  io.resin.device-type="raspberrypi"
+RUN apt-get update && apt-get install -y --no-install-recommends wget \
+	&& wget http://archive.raspberrypi.org/debian/raspberrypi.gpg.key \
```

Comment:
This is totally unsecure. You're downloading a key over HTTP and then trusting it.